docs/cql/ddl.rst

```diff
@@ -975,6 +975,8 @@ The ``ALTER TABLE`` statement can:
    Scylla allows the client to provide any timestamp on any table, it is theoretically possible to use another
    convention. Please be aware that if you do so, dropping a column will not work correctly.
 
+.. warning:: Dropping a column with timestamp specified should only be used to restore schema from description.
```

Comment:
I'm not sure an uninformed reader will understand what this warning is actually warning about. Perhaps adding specific in parentheses commands like "DROP column_name USING TIMESTAMP timestamp" and "DESC SCHEMA" will help explain what the English words "with timestamp specified" and "from description" means in practice?

Reply:
added